react-app/src/components/projects/ProjectNotificationsTable.tsx

```diff
@@ -0,0 +1,119 @@
+import { dateFormatter } from '@/utilities/formatters';
+import { Box, Typography } from '@mui/material';
+import { DataGrid, GridColDef } from '@mui/x-data-grid';
+import React, { useEffect, useState } from 'react';
+
+export interface INotificationModel {
+  id: string;
+  agency: string;
+  status: string;
+  sendOn: Date;
+  to: string;
+  subject: string;
+}
+
+interface ProjectNotificationsTableProps {
+  rows: INotificationModel[];
+  noteText?: string;
+}
+
+const ProjectNotificationsTable = (props: ProjectNotificationsTableProps) => {
+  const [paginationModel, setPaginationModel] = useState({
+    pageSize: 10,
+    page: 0,
+  });
+
+  useEffect(() => {
+    fetchData();
+  }, [paginationModel.page, paginationModel.pageSize]);
```

Comment:
Is server-side pagination being used here? It looks like it's configured to be client side.